docs/demos/xfer-gluon-with-modelhandler.ipynb

```diff
@@ -0,0 +1,582 @@
+    "    outputs = []\n",
+    "    for x in data:\n",
+    "        outputs.append(net(x))\n",
+    "    # Updates internal evaluation\n",
+    "    metric.update(label, outputs)\n",
+    "print('validation acc: {}={}'.format(*metric.get()))\n",
+    "assert metric.get()[1] > 0.94"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Convert Gluon model to Module\n",
+    "\n",
+    "*Adapted from snippet found [here](https://github.com/apache/incubator-mxnet/issues/9374)*\n",
+    "\n",
+    "From the Gluon model, the symbol and parameters are extracted and used to define an `Module` object."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "def block2symbol(block):\n",
+    "    data = mx.sym.Variable('data')\n",
+    "    sym = block(data)\n",
+    "    args = {}\n",
+    "    auxs = {}\n",
+    "    for k, v in block.collect_params().items():\n",
+    "        args[k] = mx.nd.array(v.data().asnumpy())\n",
+    "        auxs[k] = mx.nd.array(v.data().asnumpy())\n",
+    "    return sym, args, auxs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "def symbol2mod(sym, args, auxs, data_iter):\n",
+    "    mx_sym = mx.sym.SoftmaxOutput(data=sym, name='softmax')\n",
+    "    model = mx.mod.Module(symbol=mx_sym, context=mx.cpu(), \n",
+    "                          label_names=['softmax_label'])\n",
+    "    model.bind( data_shapes = data_iter.provide_data, \n",
+    "                label_shapes = data_iter.provide_label )\n",
+    "    model.set_params(args, auxs)  \n",
+    "    return model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "sym_params = block2symbol(net)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "mod = symbol2mod(*sym_params, train_data)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Alternative Method\n",
+    "\n",
+    "Serialise Gluon model to file using `.export()`.\n",
+    "\n",
+    "Load the serialised model as an MXNet Module with `Module.load()` so that xfer can be used."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# model_name = 'gluon-model'\n",
+    "# net.export(model_name)\n",
+    "\n",
+    "# mod = mx.mod.Module.load(model_name, 0, label_names=[])\n",
+    "# os.remove(model_name+'-symbol.json')\n",
+    "# os.remove(model_name+'-0000.params')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Apply ModelHandler\n",
```

Comment:
Given that it's a nice functionality of ModelHandler, consider adding a cell to drop/add layers and visualize the resulting network.